### PR TITLE
Updated daisy schematic and pcb

### LINKED ISSUE #212: Wrong Schematics for Daisy Board
Hello,
I spent 2 days debuging a daisy board that I recently ordered.
The main page is showing shematics for the daisy board with CLKSEL tied to DVDD.
So in my head the clock was internally generated. 
I had to manually test it with a multimeter to see that the board that I got have CLKSEL tied to AGND.
So please change or edit the schematics so we can see it's actually tied to AGND.
Thank you 
Valentin

### PR DESCRIPTION
**Pull request by tharun-OpenBCI**
Fixes #212

### CODE REVIEW

File: website/docs/assets/CytonImages/OBCI_V3_Daisy-Schematic.jpg
Comment (retiutut) [inline website/docs/assets/CytonImages/OBCI_V3_Daisy-Schematic.jpg:1]: Need higher resolution image here

**Review by retiutut — CHANGES_REQUESTED** [review]
Schematic is not high enough resolution to zoom in and read labels

File: website/docs/assets/CytonImages/OBCI_V3_Daisy-Schematic.jpg
Comment (tharun-OpenBCI) [reply website/docs/assets/CytonImages/OBCI_V3_Daisy-Schematic.jpg:1]: Added higher resolution schematic image

File: website/package.json [hunk 1]
```diff
@@ -13,10 +13,10 @@
     "serve": "docusaurus serve"
   },
   "dependencies": {
-    "@docusaurus/core": "2.4.3",
-    "@docusaurus/preset-classic": "2.4.3",
+    "@docusaurus/core": "^3.5.2",
```
Comment (retiutut) [inline website/package.json:16]: Please revert this change

File: website/package.json [hunk 2]
```diff
@@ -13,10 +13,10 @@
...
+    "@docusaurus/preset-classic": "^3.5.2",
     "clsx": "^1.1.1",
-    "react": "^17.0.1",
-    "react-dom": "^17.0.1"
+    "react": "^18.0.1",
```
Comment (retiutut) [inline website/package.json:19]: Please revert this change

**Review by retiutut — CHANGES_REQUESTED** [review]
Not the PR to bump the docusaurus version

File: website/package.json (hunk 1 above)
Comment (tharun-OpenBCI) [reply website/package.json]: Reverted version

File: website/package.json (hunk 2 above)
Comment (tharun-OpenBCI) [reply website/package.json]: Reverted version

**Review by retiutut — APPROVED** [review]
